agent-bootstrap/src/main/java/io/opentelemetry/auto/bootstrap/instrumentation/decorator/HttpClientDecorator.java

```diff
@@ -120,8 +120,9 @@ public Span onResponse(final Span span, final RESPONSE response) {
       if (status != null) {
         span.setAttribute(Tags.HTTP_STATUS, status);
 
-        if (Config.get().getHttpClientErrorStatuses().get(status)) {
-          span.setStatus(Status.UNKNOWN);
+        Status otelStatus = HttpUtil.statusFromHttpStatus(status);
+        if (!otelStatus.isOk()) {
```

Comment:
Why don't we set `Ok` status?

Reply:
Thought it's not helpful and would just cause extra data to be sent, but indeed this probably isn't the correct layer to worry about it (exporters could strip out OK status if they want)